api/metricsdebug/client_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package metricsdebug_test
+
+import (
+	"errors"
+	"time"
+
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/api/metricsdebug"
+	"github.com/juju/juju/apiserver/common"
+	"github.com/juju/juju/apiserver/params"
+	jujutesting "github.com/juju/juju/juju/testing"
+	"github.com/juju/juju/testing/factory"
+)
+
+type metricsdebugSuite struct {
+	jujutesting.JujuConnSuite
+
+	manager *metricsdebug.Client
+}
+
+var _ = gc.Suite(&metricsdebugSuite{})
+
+func (s *metricsdebugSuite) SetUpTest(c *gc.C) {
+	s.JujuConnSuite.SetUpTest(c)
```

Comment:
@cmars it's still needed for some of the stup in the apiserver. even though we end up mocking part of it